### ISSUE TITLE
Upgrade Wizard does not run

### ISSUE THREAD

**Issue by cnoelker** [lochmueller/calendarize#266]
Migrate the calendarize configurations to the new include/exclude/override/cutout structure creates these SQLs which are non valid:
UPDATE `tx_calendarize_domain_model_configuration` SET `type` = :dcValue1, `handling` = :dcValue2 WHERE `type` = 'timeExclude'
UPDATE `tx_calendarize_domain_model_configuration` SET `type` = :dcValue3, `handling` = :dcValue4 WHERE `type` = 'include'
UPDATE `tx_calendarize_domain_model_configuration` SET `type` = :dcValue5, `handling` = :dcValue6 WHERE `type` = 'exclude'
UPDATE `tx_calendarize_domain_model_configuration` SET `handling` = :dcValue7 WHERE (`handling` = '') OR (`handling` IS NULL)

**Comment by lochmueller** [comment]
@cnoelker Why are the SQL statements are not valid? Which SQL errors to you get?

**Comment by cnoelker** [comment]
You are right. The SQL statements are correct. The problem is that the Update Wizard is still notifying the user about 1 line which needs to be updated. That's because the counting is not implemented correctly. Replace line 48 in NewIncludeExcludeStructureUpdate.php with
$count = $q->execute()->fetchColumn(0);
=> Problem solved

**Comment by lochmueller** [comment]
Great... fixed in current master.
